apps/chat/agent/tools.py

```
@@ -451,8 +468,9 @@ class GetSessionStateTool(CustomBaseTool):
     requires_session: bool = True
     args_schema: type[schemas.GetSessionStateSchema] = schemas.GetSessionStateSchema
 
-    def action(self, key: str):
-        value = self.experiment_session.state.get(key)
+    def action(self, key: str, tool_call_id: str, graph_state: dict):
```

Ruff (0.13.1)

Unused method argument: `tool_call_id`

(ARG002)